### PR TITLE
[Changelog] Update for Hammer-1

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,13 +20,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Add post compression support [(#288)](https://github.com/ManageIQ/manageiq-appliance-build/pull/288)
 - Compress Azure, HyperV and EC2 images [(#289)](https://github.com/ManageIQ/manageiq-appliance-build/pull/289)
 - Move from apache module mod_auth_kerb to mod_auth_gssapi [(#282)](https://github.com/ManageIQ/manageiq-appliance-build/pull/282)
-- Add dracut-config-generic for EC2 ENA support [(#277)](https://github.com/ManageIQ/manageiq-appliance-build/pull/277)
```
Comment (carbonin) [inline CHANGELOG.md]: Where did this change go? How was it in beta, but not in GA?

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,13 +20,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
 
 ### Fixed
 - Update vSphere ova [(#278)](https://github.com/ManageIQ/manageiq-appliance-build/pull/278)
 - Increase filesystem sizes [(#232)](https://github.com/ManageIQ/manageiq-appliance-build/pull/232)
 - Set commit_sha just once [(#290)](https://github.com/ManageIQ/manageiq-appliance-build/pull/290)
-- Update to node 8 [(#276)](https://github.com/ManageIQ/manageiq-appliance-build/pull/276)
```
Comment (carbonin) [inline CHANGELOG.md]: Same here. Why was this not included in the GA entry?

File: CHANGELOG.md [hunk 3]
```diff
@@ -31,6 +28,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Set commit_sha just once [(#290)](https://github.com/ManageIQ/manageiq-appliance-build/pull/290)
 - Update to node 8 [(#276)](https://github.com/ManageIQ/manageiq-appliance-build/pull/276)
 
+### Removed
+- Remove non-existing "server-policy" yum group [(#268)](https://github.com/ManageIQ/manageiq-appliance-build/pull/268)
```
Comment (simaishi) [inline CHANGELOG.md]: @JPrause This is already in Gap-4 changelog, so not needed here.

**Comment by simaishi** [comment]
Hammer backport details:

```
$ git log -1
commit 220cb272ef1eae501a32239ed01a1889d0d88b01
Author: Nick Carboni <ncarboni@redhat.com>
Date:   Wed Jan 9 10:07:50 2019 -0500

    Merge pull request #304 from JPrause/hammer1_cl
    
    [Changelog] Update for Hammer-1
    
    (cherry picked from commit 155623c15c1fe41f55312aabf69ec76fa7f6a7b7)
```

**Pull request merged**